docs/check-coupled-files.md

```diff
@@ -0,0 +1,24 @@
+# Task `check-coupled-files`
+
+### What does it do?
+
+For each opened pull request it checks, whether files that usually should be changed in one changeset are actually changed.
+Think for example - whenever `package.json` changes, `package-lock.json`, should also be changed.
+If it detects missing change in one of fileset, it's reported in form of comment under pull request.
+
+Whenever newly opened pull request contains valid JIRA ID in title, it adds link to that issue to PR body.
```

Comment:
Sounds unrelated...

Reply:
Ahh.. yes, thank you, copy&paste leftovers.